credentials/alts/alts.go

```diff
@@ -91,6 +91,19 @@ type AuthInfo interface {
 	PeerRPCVersions() *altspb.RpcProtocolVersions
 }
 
+// ClientOptions contains the client-side options of an ALTS channel. These
+// options will be passed to the underlying ALTS handshaker.
+type ClientOptions struct {
+	// TargetServiceAccounts contains a list of expected target service
+	// accounts.
+	TargetServiceAccounts []string
```

Comment:
Is this always (or nearly always) supposed to be non-empty?  Or is it common to omit it?

If it's commonly used, it probably should be left as a parameter.

Reply:
This is not commonly used. I'm expecting callers will set this in very few situations.